data/human/human missions.txt

```diff
@@ -6849,6 +6849,133 @@ mission "FW Refugees to Humanika"
+	minor
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		not attributes "uninhabited"
+	to offer
+		random < 4
+		"days since start" > 30
+	on offer
+		conversation
+			`As you touch down on <origin>, you see that the landing pad next to yours holds a freighter that is practically falling apart. Standing next to it is a man with the grizzled look of an experienced captain.`
+			`	Given the sorry state of his ship he might be in need of assistance.`
+			choice
+				`	(Offer your help.)`
+				`	(Don't help him.)`
+					decline
+			`	The captain notices you approaching, and clearly guessing your intentions, speaks. "Oh don't worry, this ain't my ship."`
+			choice
+				`	"What are you doing with it then?"`
+					goto what
+				`	"Whose is it then?"`
+			`	"Well quite simply, it's nobody's."`
+			choice
+				`	"What are you doing with it then?"`
+				`	"If it isn't yours, why do you have it?"`
+			label what
+			`	An easy, broad smile upon his face tells you he's heard such a question before. "Glad you asked. It's always a pleasure to teach another captain about the niche trade of derelict hunting."`
+			choice
+				`	"Derelict hunting?"`
+					goto confused
+				`	"So you search for abandoned ships?"`
+					goto precisely
+			label confused
+			`	"Derelict hunting involves searching abandoned ship hulls. They usually have a few parts left on them that you can sell for a few credits, and if you're lucky you might even score some rare outfits or the whole hull if it's stable enough. Wouldn't recommend actually using that hull though. They're usually very fragile like this freighter here."`
+			choice
+				`	"So you search for abandoned ships and loot them?"`
+				`	"I get it, you're recycling old ships for profit."`
+			label precisely
+			`	"Precisely. If you want to go hunting for derelicts yourself, they can be rarely found in out of the way uninhabited systems. Good hunting out there." He turns back to his score.`
+				decline
+
+mission "Derelict Treasure Map"
+	description "Follow the treasure map you found in a derelict ship."
+	landing
+	to offer
+		has "derelict treasure map"
+	destination "New Argentina"
+	on offer
+		conversation
+			`Upon landing on <origin> you take out the treasure map you recovered from the derelict ship. The map gives detailed instructions about where the treasure was buried on the sparsely populated world of <destination>. However, the map's creator claims to have obtained some kind of alien weapons that disrupts a ship's electrical systems. The outlandish claim seems to bring doubt to the validity of the map.`
+			choice
+				`	(Follow the map.)`
+					accept
+				`	(Throw the map away, it's probably fake.)`
+					decline
+	on complete
+		conversation
+			`Flying low over <planet> you follow the instructions to an isolated valley where you promptly land and get to digging. After hours of shovelling the hard soil you're met with a thunk as you strike metal.`
```

Comment:
```suggestion
			`Flying low over <planet> you follow the instructions to an isolated valley where you promptly land and get to digging. After hours of shovelings the hard soil you're met with a thunk as you strike metal.`
```

Reply:
I assume you just meant to drop an l and not add the s?

Reply:
Yeah.